src/stdlib_stats_distribution_normal.fypp

```diff
@@ -62,272 +57,264 @@ module stdlib_stats_distribution_normal
     !! cdf_normal-normal-distribution-cumulative-distribution-function))
     !!
         #:for k1, t1 in RC_KINDS_TYPES
-        module procedure cdf_norm_${t1[0]}$${k1}$
+            module procedure cdf_norm_${t1[0]}$${k1}$
         #:endfor
     end interface cdf_normal
 
-
-
-
-
 contains
 
-    subroutine zigset
-    ! Marsaglia & Tsang generator for random normals & random exponentials.
-    ! Translated from C by Alan Miller (amiller@bigpond.net.au), released as public
-    ! domain (https://jblevins.org/mirror/amiller/)
-    !
-    ! Marsaglia, G. & Tsang, W.W. (2000) `The ziggurat method for generating
-    ! random variables', J. Statist. Software, v5(8).
-    !
-    ! This is an electronic journal which can be downloaded from:
-    ! http://www.jstatsoft.org/v05/i08
-    !
-    ! Latest version - 1 January 2001
-    !
+    impure subroutine zigset
```

Comment:
This *impure* attribute is redundant. I would suggest to remove it in this case and other occurrences.

Reply:
That deserves a bit of explanation. IMO, the attribute is not redundant from a "documentation" perspective.
A function without an explicit attribute will be treated as impure, but it may in reality be pure or impure. The only way to find out it is to look inside the code. Adding the impure statement there is meant to explicitly indicate that that function is really impure (it changes global variables).

Reply:
I think it's okay for `impure` to stay for documentation purposes.